### ISSUE TITLE
Stage does not return to initial position after cutting

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/BakingTray#195]
Very odd. Some recent change must have created this problem. Sometimes it takes two cuts before it returns to the initial position. Other times it gets a little closer the second time but it's still not in the same position.

**Comment by raacampbell** [comment]
The problem stems from the fact that slice sample reads a different _y_ stage position to that displayed in the GUI. It faithfully returns to this different position. What is really odd is that I, say, move to y=1.23 then at the command line I do:
```
>> hBT.getXYpos
X=26.50, Y=1.23
```
Fine. But then when I cut I get|"
```
Position before cutting: x=26.50 y=0.79
```

**Comment by raacampbell** [comment]
Ah. Fixed: 042d39c229e619387b546dc5cbf27d5694f0429b
Pressing the slice sample button was telling the stage to move to _y=0_ before cutting. Removed that and we're good now.
